app/workers/store_builds.rb

```diff
@@ -73,7 +73,11 @@ def artifact_s3_path(uri, build)
     filename = uri.path.split('/').last;
 
     project_name = "#{build['username']}_#{build['reponame']}"
-    suite_name = "SUITE=#{uri.path.split('/')[-2]}"
+
+    suite = uri.path.split('/')[-2]
+    suite = build['reponame'] if suite == "circle-junit"
```

Comment:
right now we rely on the artifacts being in the circle-junit folder. so this is basically saying if all of the artifacts are in the circle-junit folder, just name the suite reponame, rather than circle-junit